src/itype.h

```diff
@@ -257,7 +260,7 @@ enum class encumbrance_modifier_type : int {
 struct armor_portion_data {
 
     // The base volume for an item
-    const units::volume volume_per_encumbrance = 250_ml; // NOLINT(cata-serialize)
+    static constexpr units::volume volume_per_encumbrance = 250_ml; // NOLINT(cata-serialize)
```

Comment:
In C++14 you need to define
```cpp
constexpr units::volume armor_portion_data::volume_per_encumbrance;
```
outside `struct armor_portion_data`.

Reply:
I can't find an example of another constexpr that does this somewhere else in the code so I'm not sure where it needs to be defined.